src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/resource/ReadinessProfileTestCase.kt

```diff
@@ -24,7 +24,7 @@ abstract class ReadinessProfileTestCase : IntegrationTestBase() {
     expectedStatus: HttpStatus = HttpStatus.CREATED,
   ): ResponseEntity<ReadinessProfileDTO> {
     val result = restTemplate.exchange(
-      "/readiness-profiles/$prisonNumber",
+      "$URL_PREFIX_OF_READINESS_PROFILE/$prisonNumber",
```

Comment:
Can we have instead, just to save the "OF" particle, "$READINESS_PROFILE_URL_PREFIX" or, even better, "READINESS_PROFILE_ENDPOINT"

Reply:
updated